app/models/trainee.rb

```diff
@@ -186,8 +188,14 @@ class Trainee < ApplicationRecord
   scope :with_manual_application, -> { where(apply_application: nil, created_from_dttp: false, hesa_id: nil) }
   scope :with_apply_application, -> { where.not(apply_application: nil) }
   scope :created_from_dttp, -> { where(created_from_dttp: true) }
+  
+  # We only look for the HESA ID to determine if a trainee record came from HESA.
+  # Even though some records imported from DTTP will have a HESA ID, their original source is HESA so we chose this implementation
```

Comment:
i've added a comment here to clarify why we chose to only use HESA ID to identify HESA records as I thought the business logic wasn't necessarily clear - can remove if this is not right/useful